src/main/model/JWTHeader.java

```diff
@@ -0,0 +1,58 @@
+package main.model;
+//model for: JWT=$(hmcOktaJWT.py -o cnqr.okta.com -H 'HMC - US2' -e p00849947wds -R 22717526)
+public class JWTHeader {
+    private String PythonCall;
+    private String url;
+    private String dataCenter;
```

Comment:
Does this come out as ``HMC - US2``? That seems odd, and datacenter seems to not quite be what you're looking for as a name. To what does ``HMC - US2`` refer?